crates/uv-toolchain/fetch-version-metadata.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python3.12
```

Comment:
Nearly unchanged from the `scripts/boostrap` script which will be deleted in the near future.